lxd/device/nic_macvlan.go

```diff
@@ -70,7 +71,7 @@ func (d *nicMACVLAN) validateConfig(instConf instance.ConfigReader) error {
 		d.config["parent"] = netConfig["parent"]
 
 		// Copy certain keys verbatim from the network's settings.
-		inheritKeys := []string{"mtu", "vlan", "maas.subnet.ipv4", "maas.subnet.ipv6"}
+		inheritKeys := []string{"mtu", "vlan", "maas.subnet.ipv4", "maas.subnet.ipv6", "gvrp"}
```

Comment:
@TimRots although you have specified that `gvrp` key should be inherited from the macvlan managed network when using the `network` device property, you've not yet allowed the `gvrp` property to be set on the `macvlan` managed network type, and this means that it will be impossible to inherit from it.

Please can you add it here to0 https://github.com/lxc/lxd/blob/master/lxd/network/driver_macvlan.go#L29

Reply:
Done, also applied your other comment from the closed pr